Libs/MRML/Core/vtkMRMLStorageNode.h

```diff
@@ -26,6 +27,77 @@ class vtkStringArray;
 
 // STD includes
 #include <vector>
+#include <algorithm>
+
+///
+/// A class for recording warnings and errors associated with this
+/// vtkMRMLStorableNode.
+class VTK_MRML_EXPORT vtkMessageCollection : public vtkObject
+{
+public:
+  vtkTypeMacro(vtkMessageCollection,vtkObject);
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+  static vtkMessageCollection* New();
+
+  ///
+  /// A helper class that describes a single message
+  struct Message
```

Comment:
This should  probably be maintained in the private implementation.

Reply:
I am not sure that I understand.  `vtkMessageCollection::Message` is a return value for a public method and would need to be public.